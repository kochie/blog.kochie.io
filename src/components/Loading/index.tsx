import React, { ReactElement } from 'react'

const Loading = (): ReactElement => {
  return (
    <div>
      <p>Loading...</p>
    </div>
  )
}

export default Loading
