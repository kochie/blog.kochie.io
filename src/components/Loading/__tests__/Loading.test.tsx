import React from 'react'
import { describe, test, expect } from 'vitest'

import Loading from '@/components/Loading'
import { render } from '@testing-library/react'

describe('LOADING COMPONENT', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Loading />)

    expect(asFragment()).toMatchSnapshot()
  })
})
