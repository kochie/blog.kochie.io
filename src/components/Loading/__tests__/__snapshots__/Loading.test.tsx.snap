// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LOADING COMPONENT > renders correctly 1`] = `
<DocumentFragment>
  <div>
    <p>
      Loading...
    </p>
  </div>
</DocumentFragment>
`;
